storybook/src/components/Breakout/Breakout.docs.mdx

```diff
@@ -30,6 +30,12 @@ On narrower screens, let the text move below the image.
 
 ## Examples
 
+### With caption
+
+To add a caption to the image, wrap both in a Figure.
+
+<Canvas of={BreakoutStories.WithCaption} />
```

Comment:
This example shows an interesting issue imo. The text in a breakout isn't a caption? Or it sometimes is? It's a little confusing I think.

Reply:
Yes, let’s think this through later: https://gemeente-amsterdam.atlassian.net/browse/DES-1121. I’ve removed the example.